test/unit/authentication/DPoPWebIdExtractor.test.ts

```diff
@@ -0,0 +1,112 @@
+import { verify } from 'ts-dpop';
+import { DPoPWebIdExtractor } from '../../../src/authentication/DPoPWebIdExtractor';
+import { TargetExtractor } from '../../../src/ldp/http/TargetExtractor';
+import type { ResourceIdentifier } from '../../../src/ldp/representation/ResourceIdentifier';
+import type { HttpRequest } from '../../../src/server/HttpRequest';
+
+class DummyTargetExtractor extends TargetExtractor {
+  public async handle(): Promise<ResourceIdentifier> {
+    return { path: 'http://example.org/foo/bar' };
+  }
+}
+
+describe('A DPoPWebIdExtractor', (): void => {
+  const targetExtractor = new DummyTargetExtractor();
```

Comment:
You can use the utility class `new StaticAsyncHandler(true, { path: 'http://example.org/foo/bar' });` here.